modules/ROOT/pages/security/ip-filtering.adoc

```diff
@@ -94,3 +94,15 @@ With IP filtering:
 To use `neo4j-admin database upload` add the specific link:https://support.neo4j.com/s/article/360050504254-What-are-the-public-IP-addresses-to-provision-in-a-firewall-configuration-to-allow-Aura-use[Control Plane Egress] to your IP filter allow list. 
 This ensures the temporary connection needed to push data is permitted.
 
+== Aura API
+
+The link:https://neo4j.com/docs/aura/platform/api/specification/?urls.primaryName=Aura%20v2beta1#/[Aura API `v2beta1`]:
```

Comment:
IP filtering can also be achieved via API, as described in link:{neo4j-docs-base-uri}/aura/platform/api/specification/?urls.primaryName=Aura%20v2beta1#/[Aura API specification].
Note that this is only available with v2beta1.